docs/sdk/api/apiRequestBuilder.md

```diff
@@ -14,27 +14,31 @@ npm install --save @commercetools/api-request-builder
 <script>// global: CommercetoolsApiRequestBuilder</script>
 ```
 
-## `createRequestBuilder(customServices)`
+## `createRequestBuilder(projectKey, customServices)`
```

Comment:
Sorry, now that I see this I think it's better to have named arguments

```js
// before
createRequestBuilder(projectKey, customServices)

// after
createRequestBuilder(options)
// options -> { projectKey, customServices }
```

/cc @hisabimbola ?